JustSaying/JustSayingFluentlyExtensions.cs

```diff
@@ -12,6 +14,8 @@ public static class JustSayingFluentlyExtensions
     {
         public static IMayWantOptionalSettings InRegion(this JustSayingFluentlyLogging logging, string region) => InRegions(logging, region);
 
+        public static IMayWantOptionalSettings InRegions(this JustSayingFluentlyLogging logging, IEnumerable<string> regions) => InRegions(logging, regions.ToArray());
```

Comment:
Does it work OK with overload resolution if the implementation is in ```IEnumerable<string>``` and the ```string[]``` overload calls it? Then we don't need the ```ToArray()``` call.